src/ralph/ssl_certificates/management/commands/import_ssl_certificates.py

```diff
@@ -11,23 +11,88 @@
 from django.core.management.base import BaseCommand
 
 from ralph.assets.models.assets import Manufacturer
-from ralph.ssl_certificates.models import SSLCertificate
+from ralph.ssl_certificates.models import CertificateType, SSLCertificate
 
 
-class Command(BaseCommand):
+DEFAULT_ISSUER_NAME = 'CA ENT'
+
+
+def extract_domain_from_filename(filename):
+    """
+    >>> extract_domain_from_filename('wildcard_allegro.pl.crt')
+    wildcard_allegro.pl
+    """
+    return Path(filename).stem
+
+
+def get_domain_ssl(cert):
+    domain_subject = cert.subject.get_attributes_for_oid(NameOID.COMMON_NAME)
+    domain_ssl = domain_subject[0].value
+    if domain_ssl.startswith('*.'):
+        domain_ssl = domain_ssl[2:]
+    return domain_ssl
+
+
+def get_ssl_type(issuer_name, san, filename):
+    type_ssl = CertificateType.ov.id
+    if re.match(r'^(wildcard.+)', filename):
+        type_ssl = CertificateType.wildcard.id
+    elif san is not '':
+        type_ssl = CertificateType.multisan.id
+    elif issuer_name == DEFAULT_ISSUER_NAME:
+        type_ssl = CertificateType.internal.id
+    return type_ssl
 
+
+class Command(BaseCommand):
     help = 'Import data to application from dir'
 
     def add_arguments(self, parser):
         parser.add_argument('certs_dir', type=str)
 
+    def get_san(self, cert):
+        san = ''
+        extension = None
+        try:
+            extension = cert.extensions.get_extension_for_oid(
+                ExtensionOID.SUBJECT_ALTERNATIVE_NAME
+            )
+        except ExtensionNotFound:
+            pass
+        if extension and extension.value:
+            san = extension.value.get_values_for_type(x509.DNSName)
+        return san
+
+    def ssl_values(self, cert, san, filename):
```

Comment:
could the name of this method be more descriptive (+ adding some docstring what happens here would be nice too)